airflow/cli/cli_parser.py

```diff
@@ -516,18 +516,32 @@ def string_list_type(val):
     type=int,
     default=60,
 )
-ARG_DB_VERSION = Arg(
+ARG_DB_UG_VERSION = Arg(
     (
         "-n",
         "--version",
     ),
-    help="The airflow version to downgrade to. Note: must provide either `--revision` or `--version`.",
+    help="(Optional) The airflow version to upgrade to. Note: must provide either `--revision` or `--version`.",
+)
+ARG_DB_UG_REVISION = Arg(
```

Comment:
NIT: Why not UPGRADE/DOWNGRADE verbatim?

Reply:
Yup you're right that's better